### PR TITLE
PS-4386 : Fix various places where macros are redefined or used when …

### PR DESCRIPTION
**Pull request by george-lorch**
…undefined

- Fix various places where macros are redefined or used when undefined-

### CODE REVIEW

**Comment by george-lorch** [comment]
https://jenkins.percona.com/view/PS%205.6/job/percona-server-5.6-param/2206/

**Comment by george-lorch** [comment]
new jenkins https://ps.cd.percona.com/view/5.6/job/percona-server-5.6-param/10/

File: storage/tokudb/tokudb_sysvars.h [hunk 1]
```diff
@@ -93,10 +93,10 @@ extern my_bool      gdb_on_fatal;
 
 extern my_bool         check_jemalloc;
 
-#if TOKUDB_DEBUG
+#if defined(TOKUDB_DEBUG) && TOKUDB_DEBUG
```
Comment (laurynas-biveinis) [inline storage/tokudb/tokudb_sysvars.h]: Now there are two forms of this check: above defined(FOO) and here defined(FOO) && FOO. Should be a single form (IMHO the former one) only

**Review by laurynas-biveinis — CHANGES_REQUESTED** [review]

File: storage/tokudb/tokudb_sysvars.h (hunk 1 above)
Comment (george-lorch) [reply storage/tokudb/tokudb_sysvars.h]: Ahh, yeah, good point.

**Comment by george-lorch** [comment]
Fixed and new jenkins https://ps.cd.percona.com/view/5.6/job/percona-server-5.6-param/12/

**Review by laurynas-biveinis — APPROVED** [review]

**Pull request merged**